appveyor.yml

```diff
@@ -1,17 +1,18 @@
-os: Visual Studio 2022
+os: Ubuntu2204
 
-before_build:
-  - nuget restore Renci.SshNet.sln
+services:
+- docker
 
-install:
-  - cinst dotnet-sdk --version=7.0.403 --limit-output
-
-build:
-  project: Renci.SshNet.sln
-  verbosity: minimal
+#for testing purposes only
+build_script:
+  - dotnet build test/Renci.SshNet.IntegrationTests/ -c Debug
 
 test_script:
-- cmd: >-
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net462\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
-    
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net7.0\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
+- mkdir artifacts
+- sudo tcpdump -i any -c1000 -w artifacts/tcpdump.pcap
+#- dotnet test -c Debug --no-restore --no-build --results-directory artifacts --logger "console;verbosity=detailed" --logger "liquid.md;LogFileName=TestReport.md" -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura -p:CoverletOutput=artifacts/coverage.xml test/Renci.SshNet.IntegrationTests/Renci.SshNet.IntegrationTests.csproj
```

Comment:
```suggestion
- sudo tcpdump -i any -c1000 -w artifacts/tcpdump.pcap -v
- dotnet test -c Debug --no-restore --no-build --results-directory artifacts --logger "console;verbosity=detailed" --logger "liquid.md;LogFileName=TestReport.md" -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura -p:CoverletOutput=artifacts/coverage.xml test/Renci.SshNet.IntegrationTests/Renci.SshNet.IntegrationTests.csproj
```

Reply:
tcpdump waits for the end of this command. It does not work in background.